api/run_psql_client.sh

```diff
@@ -0,0 +1,2 @@
+#! /bin/sh
+docker run  --network api_default -it -e PGPASSWORD=mysecretpassword --rm --link api_postgres_1:postgres postgres:9.6.6 psql -h postgres -U postgres -d postgres
```

Comment:
Is it possible to use `docker-compose` for this? I think `api_postgres_1` might change depending on the computer?

Reply:
Ahhh, yes, awesome idea! I'll do that!